src/AppInstallerCLICore/Workflows/InstallFlow.cpp

```diff
@@ -128,6 +142,19 @@ namespace AppInstaller::CLI::Workflow
         }
     }
 
+    void CheckForUnsupportedArgs(Execution::Context& context)
+    {
+        const auto& unsupportedArgs = context.Get<Execution::Data::Installer>()->UnsupportedArguments;
+
+        for (auto unsupportedArg : unsupportedArgs)
+        {
+            if (context.Args.Contains(GetUnsupportedArgumentType(unsupportedArg)))
+            {
+                context.Reporter.Warn() << Resource::String::UnsupportedArgument << " --"_liv << UnsupportedArgumentToString(unsupportedArg) << std::endl;

```

Comment:
If `--log` is an unsupported argument, and a user uses `winget install <Package> -o <Path>` then the warning would be `This argument is not supported for this package: --log` which could be confusing to some users. I would suggest adding some logic to determine if its an unsupported argument or an unsupported alias which was used

Reply:
I think including the name and alias here would be just as effective of a solution

Reply:
Changed the current code to find the actual `Argument` rather than just checking the `Args::Type`